syncer/syncer.go

```diff
@@ -417,6 +448,25 @@ func (s *Syncer) getATXs(ctx context.Context, layerID types.LayerID) error {
 	return nil
 }
 
+func (s *Syncer) processTortoiseBeacons(ctx context.Context, layer *types.Layer) {
+	for _, b := range layer.Blocks() {
+		if _, ok := s.seenTB[b.MinerID().String()]; ok {
+			// TODO(nkryuchkov): invalid block, miner attempted to include beacon in two blocks within the same epoch
+		}
+
+		s.seenTB[b.MinerID().String()] = struct{}{}
+
+		tb := b.EligibilityProof.TortoiseBeacon
+		tbString := string(tb)
+
+		s.tbCount[tbString]++
+		if s.tbCount[tbString] > s.mostUsedTBCount {
```

Comment:
how is the first TB defined? where does the value come from?

Reply:
Do you mean the initial value of `mostUsedTBCount`?